doc/nrf/releases/release-notes-latest.rst

```diff
@@ -91,6 +91,79 @@ The following list summarizes the most important changes inherited from upstream
 
   * Removed the ``net_idx`` parameter from the Health Client model APIs because it can be derived (by the stack) from the ``app_idx`` parameter.
 
+* Drivers:
+
+  * Clock control:
+
+    * Fixed an issue in the nRF clock control driver that could lead to a fatal error during the system initialization, when calibration was started before kernel services became available.
+
+  * Display:
+
+    * Added support for temperature sensors in the SSD16xx driver.
+
+  * Entropy:
+
+    * Fixed a race condition in the nRF5 entropy driver that could result in missing the wake-up event (which caused the ``kernel.memory_protection.stack_random`` test to fail).
+
+  * Flash:
+
+    * Extended the flash API with the :cpp:func:`flash_get_parameters` function.
+    * Fixed an issue in the Nordic Semiconductor nRF flash driver (soc_flash_nrf) that caused operations to fail if a Bluetooth central had multiple connections.
+    * Added support for a 2 IO pin setup in the nRF QSPI NOR flash driver (nrf_qspi_nor).
+    * Added support for sub-word lengths of read and write transfers in the nRF QSPI NOR flash driver (nrf_qspi_nor).
+    * Improved the handling of erase operations in the nRF QSPI NOR flash driver (nrf_qspi_nor), the AT45 family flash driver (spi_flash_at45), and the SPI NOR flash driver (spi_nor).
+      Now the operation is not started if it cannot be completed successfully.
+
+  * GPIO:
+
+    * Removed deprecated API functions and macros.
+    * Improved allocation of GPIOTE channels in the nRF GPIO driver (gpio_nrfx).
+
+  * I2C:
+
+    * Fixed handling of scattered transactions in the nRF TWIM nrfx driver (i2c_nrfx_twim) by introducing an optional concatenation buffer.
+    * Used a time limit (100 ms) when waiting for transactions to complete, in both nRF drivers.
+
+  * LoRa:
+
+    * Added support for SX126x transceivers.
+
+  * PWM:
+
+    * Clarified the expected API behavior regarding zero pulse length and non-zero pulse equal to period length.
+
+  * Sensors:
+
+    * Added support for the IIS2DH accelerometer.
```

Comment:
```suggestion
    * Added support for the LIS2DH accelerometer.
```

Reply:
IIS2DH is the one that has been added (see https://github.com/zephyrproject-rtos/zephyr/pull/25018). LIS2DH was already supported.

Reply:
Ah, thanks.